blocks/banner/banner.css

```diff
@@ -0,0 +1,180 @@
+.banner .banner-picture {
+  position: relative;
+  height: 375px;
+  padding: 0 16px;
+  max-width: 1200px;
+  background-size: cover;
+  background-position: center;
+}
+
+.banner .banner-element {
+  padding-bottom: 25px;
+  background-size: cover;
+  background-position: center;
+}
+
+.banner .spotlight-link {
+  color: inherit;
+}
+
+.banner .spotlight-text .title {
+  font-size: 42px;
+  font-weight: 900;
+  line-height: 1em;
+  padding-bottom: 20px;
+  font-family: var(--heading-font-family);
+}
+
+.banner .spotlight-text .subtitle {
+  font-size: 36px;
+  font-weight: 900;
+  line-height: 1em;
+  padding-bottom: 20px;
+  font-family: var(--heading-font-family);
+}
+
+.banner .spotlight-button {
+  background-color: var(--btn-bg-color-3);
+  color: var(--background-color);
+  margin-bottom: 20px;
+}
+
+.banner .spotlight-form {
+  height: 30px;
+}
+
+.banner .spotlight-input {
+  margin: 0%;
+  height: 30px;
+  box-sizing: border-box;
+  padding: 5px 10px;
+  font-size: var(--body-font-size-xs);
+  line-height: 1.5;
+  border: 2px solid var(--dark-color);
+}
+
+.banner .spotlight-input:focus {
+  border-color: var(--input-focus-border);
+  outline: 0;
+  box-shadow: inset 0 1px 1px rgba(0 0 0 / 7.5%), 0 0 8px rgba(102 175 233 /60%);
+}
+
+.banner .spotlight-form-button {
+  background-color: var(--yellow);
+  color: var(--dark-blue);
+  font-size: var(--body-font-size-xs);
+  box-sizing: border-box;
+  border-radius: 0 4px 4px 0;
+  max-height: 30px;
+}
+
+.banner .spotlight-circle-holder {
+  max-height: 375px;
+  box-sizing: border-box;
+  overflow: hidden;
+  position: relative;
+}
+
+.banner .spotlight-circle-holder .spotlight-circle {
+  border-radius: 288px;
+  height: 530px;
+  margin: auto;
+  border: 10px solid white;
+  transform: translateY(-15%);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.banner .spotlight-text {
+  text-align: center;
+  padding: 0 10px;
+  margin: auto;
+}
+
+.banner .mobile-bannertext-holder {
+  padding: 20px;
+}
+
+.banner-wrapper:nth-child(odd) .spotlight-text .title {
```

Comment:
The .banner-wrapper should be scoped I think under .banner

Reply:
<img width="654" alt="Screenshot 2024-05-27 at 10 05 42 AM" src="https://github.com/aemsites/hubblehomes-com/assets/6240431/242e0284-90a5-4602-93df-070f6b8c9e9e">

banner-wrapper appears before the banner class, hence putting it under .banner scope is not working as expected.

Reply:
I don't know why we need to go outside of the banner for this.  I would assume it's kind-of bad practice to be extending outside of the block itself.   It should still work ifyou use `banner:nth()`

Reply:
If I understand correctly, the nth child selector operates with sibling nodes. Banner wrappers are siblings, sharing the same parent. However, since the .banner divs are not siblings, the selector doesn't apply to them.


I tried using .banner instead of .banner-wrapper, however the CSS did not work as intended. 

<img width="536" alt="Screenshot 2024-05-27 at 2 25 05 PM" src="https://github.com/aemsites/hubblehomes-com/assets/6240431/dbce660e-cd00-4524-b7b7-7d70bc8de8a4">

<img width="1117" alt="Screenshot 2024-05-27 at 2 27 13 PM" src="https://github.com/aemsites/hubblehomes-com/assets/6240431/0b7df2a8-cd1c-4352-ba03-d2188a3e6ee3">